documents/specs/Elements.txt

```diff
@@ -584,6 +585,10 @@ k⋎                                   = 2 ** 30
 øṀ (a: string)                       = vertical_mirror(a, mapping=flip_slashes_and_brackets) # [ <-> ], ( <-> ), { <-> }, < <-> >, / <-> \
    (otherwise)                       = vectorises
 ø¦ (a: string, b: string)            = vertical_mirror(a, mapping=b) # mapping in the form of ["normal", "flipped"] - custom mapping
+øb (a: string)                       = bracketify(a)
+øe (a: string, b: string)            = enclose b in a
```

Comment:
will this also mirror?

will enclosing "Hello" in `{" be "{Hello}" or "{Hello{"?